src/theme.js

```diff
@@ -62,48 +57,30 @@ const colors = {
   text: {
     main: '#3c3e42',
     light: '#8f9297'
+  },
+
+  gray: {
+    light: '#f4f4f4',
+    main: '#c6c8c9',
+    dark: '#9f9f9f'
   }
 }
 
-export const breakpoints = [
-  32,
-  48,
-  64,
-  80
-]
+export const breakpoints = [32, 48, 64, 80]
 
-export const space = [
-  0,
-  4,
-  8,
-  16,
-  32,
-  64,
-  128,
-]
+export const space = [0, 4, 8, 16, 32, 64, 128]
 
-export const fontSizes = [
-  12,
-  14,
-  16,
-  20,
-  24,
-  32,
-  48,
-  64,
-  72,
-  96
-]
+export const fontSizes = [12, 14, 16, 20, 24, 32, 48, 64, 72, 96]
 
-export const weights = [
-  400,
-  700
-]
+export const weights = [400, 700]
 
 export const radius = 4
 export const font = `Roboto, Arial, sans-serif`
 export const monospace = `'Ubuntu Mono Web', 'Courier New', monospace`
 
+export const defaultControlHeight = 36
```

Comment:
Would be nice if this was derived from `theme.space` just to keep the theme api v simple.